wrappers/gdiplus/CMakeLists.txt

```diff
@@ -16,18 +16,19 @@ if (${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
         set (CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} /MTd")
         set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /MTd")
     endif()
-endif()
 
-if (NOT DEFINED BUILD_WRITERS)
-    set (BUILD_WRITERS OFF)
-endif()
+    if (NOT DEFINED BUILD_WRITERS)
+        set (BUILD_WRITERS OFF)
+    endif()
 
-if (NOT DEFINED BUILD_READERS)
-    set (BUILD_READERS ON)
-endif()
+    if (NOT DEFINED BUILD_READERS)
+        set (BUILD_READERS ON)
+    endif()
 
-add_subdirectory (${CMAKE_CURRENT_SOURCE_DIR}/../../core ${CMAKE_BINARY_DIR}/ZXing)
+    add_subdirectory (${CMAKE_CURRENT_SOURCE_DIR}/../../core ${CMAKE_BINARY_DIR}/ZXing)
```

Comment:
This would also need be to changed to link against the core lib instead of including it as source here. Similar to the python wrapper. That would be the 'right' way to do it, in my opinion ;).

Reply:
Agreed, you are right.

Reply:
Oups, that doesn't work. I got: 
CMake Error at core/CMakeLists.txt:438 (add_library):
  add_library cannot create target "ZXing" because another target with the
  same name already exists.  The existing target is a static library created
  in source directory "D:/Repo/zxing-cpp/core".  See documentation for
  policy CMP0002 for more details.

which seems logical since the master CMakeLists.txt already include the 'core' module.

Reply:
I think you read my comment as I intended it but my comment was quite the opposite of 'right'. I only saw the diff and did not realize it was inside this "top level project"-guard. So with the top-level build option for the wrapper inside a WIN32 block and defaulted to `ON` this compiles now on windows? -> merge?